src/main/webapp/resources/js/components/ant.design/PagedTable/PagedTableContext.js

```diff
@@ -134,11 +148,24 @@ function PagedTableProvider({
         current,
         order,
         column: field,
-        filters: pickBy(filters)
-      }
+        filters: pickBy(filters),
+      },
     });
   };
 
+  React.useMemo(() => {
```

Comment:
Can we change this to the following:

```jsx
const paginationOptions = React.useMemo(() => getPaginationOptions(state.total), [state.total]);
```

and then in `<Provider` below change it to:

```jsx
pagination: {paginationOptions},
```